code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -15,6 +15,7 @@
 	inverse_chem_val = 0.1
 	inverse_chem = null
 	failed_chem = /datum/reagent/consumable/nutriment
+	abstract_type = /datum/reagent/consumable
```

Comment:
I dont believe this is valid unless I misunderstand the interpretation, as Nutriment is a valid, non-abstract reagent used in lots of recipes, aswell as being a raw 'nutrition reagent' for food to provide hunger, so correct me if I am wrong, this should have description and be non-abstract.

Reply:
This is on `/datum/reagent/consumable` which is excluded in the legacy blacklist anyways.